### PR TITLE
feat: add blueprint code to the blueprint detail screen

### PR DESCRIPTION
**Pull request by pedroferreira1**
### Acceptance Criteria
- We must show the blueprint source code in the blueprint detail screen.

Note I: I decided to install a package to highlight the code because the UI is improved a lot, the package is extensively used, and the explorer is not a code as critical as the wallet in terms of handling sensitive data.

Note II: it needs a new API in the nano hathor core (https://github.com/HathorNetwork/nano-hathor-core/pull/96) and the corresponding API in the explorer service (https://github.com/HathorNetwork/hathor-explorer-service/pull/333)

https://github.com/user-attachments/assets/77dbd9af-08da-4fea-990e-b94f0c719e2a

### Security Checklist
- [x] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "d3-selection": "3.0.0",
     "d3-zoom": "3.0.0",
     "font-awesome": "4.7.0",
+    "highlight.js": "^11.10.0",
```
Comment (tuliomir) [inline package.json:17]: chore: Pin the version of the new dependency to stay on the current versioning pattern for this repo
```suggestion
    "highlight.js": "11.10.0",
```

File: package-lock.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
         "d3-selection": "3.0.0",
         "d3-zoom": "3.0.0",
         "font-awesome": "4.7.0",
+        "highlight.js": "^11.10.0",
```
Comment (tuliomir) [inline package-lock.json:18]: chore: Pin the version on the lockfile as well

**Review by tuliomir — DISMISSED** [review]

**Review by r4mmer — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (pedroferreira1) [reply package.json]: Done https://github.com/HathorNetwork/hathor-explorer/pull/293/commits/40aba4857a7fec3680eddd1f6365dee1c2766d82

File: package-lock.json (hunk 2 above)
Comment (pedroferreira1) [reply package-lock.json]: https://github.com/HathorNetwork/hathor-explorer/pull/293/commits/40aba4857a7fec3680eddd1f6365dee1c2766d82

**Review by tuliomir — APPROVED** [review]

**Review by r4mmer — APPROVED** [review]

**Pull request merged**